bash-completion-config.cmake.in

```diff
@@ -4,8 +4,11 @@
 set (BASH_COMPLETION_VERSION "@VERSION@")
 
 set (BASH_COMPLETION_PREFIX "@prefix@")
-set (BASH_COMPLETION_COMPATDIR "@compatdir@")
-set (BASH_COMPLETION_COMPLETIONSDIR "@pkgdatadir@/completions")
-set (BASH_COMPLETION_HELPERSDIR "@pkgdatadir@/helpers")
+set (BASH_COMPLETION_DATADIR "@datadir@")
+set (BASH_COMPLETION_SYSCONFDIR "@sysconfdir@")
+
```

Comment:
I wonder if this does what it intends to do. It introduces BASH_COMPLETION_DATADIR and BASH_COMPLETION_SYSCONFDIR, but they're not used for anything that I can tell, one cannot use them to override install locations because the definitions below don't use them; they have similarly hardwired/expanded paths as they used to.

Could you clarify the intent here?

Reply:
Totally my fault sorry due to my lack of knowledge of `cmake`. I thought that it was related to the `bash-completion.pc.in` file and that it also needed to have these two variables defined. I have removed these two lines from the commit.